apps/src/templates/teacherDashboard/RosterDialog.jsx

```diff
@@ -152,6 +156,7 @@ class RosterDialog extends React.Component {
   state = {selectedId: null};
 
   importClassroom = () => {
+    this.recordSectionSetupExitEvent(COMPLETED_EVENT);
```

Comment:
Do you know if this will also happen when a teacher syncs their section?

Reply:
hmmmmm this is an excellent question and I have no idea how syncing works. Will investigate!

Reply:
From what I can tell, syncing happens from the manage students tab, through SectionActionDropdown, via importOrUpdateRoster, which comes from teacherSectionsRedux. Theres a fetchJSON call from here. If I understand this correctly, then teachers syncing from their manage students tab won't log an event because they're not actually using this flow, right? Whereas if they were to go back through section setup again (sometimes suggested by our support article if a teacher needs to see a list of all their clever/google classrooms), yes it would log again.

Reply:
That makes sense! You can also sync from the homepage but that also doesn't bring up the dialog. 

<img width="1202" alt="Screen Shot 2023-01-13 at 4 41 34 PM" src="https://user-images.githubusercontent.com/46464143/212424514-cb2f11d8-d950-4aba-9486-a01dd3405c63.png">